ecosystem/typescript/sdk/src/plugins/token_client.ts

```diff
@@ -10,16 +10,8 @@ import * as Gen from "../generated/index";
 import { HexString, MaybeHexString } from "../utils";
 import { TransactionBuilder, TransactionBuilderRemoteABI, TxnBuilderTypes } from "../transaction_builder";
 import { MAX_U64_BIG_INT } from "../bcs/consts";
-import { TOKEN_TRANSFER_OPT_IN } from "../abis";
```

Comment:
I think this is the only const in this file, can we delete the file...?

Reply:
You mean removing the `abis.ts` file? Yes!